eslint 1.10.3 keeps bugging me about incorrect identation (but it is correct)

I'm not sure if I missed some option in .eslintrc.json, but this is what I get with the following test files:

t.js:
```javascript
'use strict';

function A() {
    this.a = function() {
        'test';
        'test';
        return 'a';
    };

    this.b = function() {
        return this.a();
    };

    this.c = {
        aa: function() {
            'test1';
            'test1';
            return 'aa';
        }
        , bb: function() {
            return this.bb();
        }
    };
}

module.exports = new A();
```

.eslintrc.json:
```json
{
    "rules": {
        "indent": [
            1,
            4
        ],
        "quotes": [
            2,
            "single"
        ],
        "linebreak-style": [
            2,
            "unix"
        ],
        "semi": [
            2,
            "always"
        ]
    },
    "env": {
        "es6": false,
        "node": true
    },
    "extends": "eslint:recommended"
}
```

output:
```bash
[wish@wish wish]$ eslint ./t.js 

/home/wish/t.js
  21:13  warning  Expected indentation of 14 space characters but found 12  indent
  22:9   warning  Expected indentation of 10 space characters but found 8   indent

✖ 2 problems (0 errors, 2 warnings)
```

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/29240438-eslint-1-10-3-keeps-bugging-me-about-incorrect-identation-but-it-is-correct?utm_campaign=plugin&utm_content=tracker%2F282608&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F282608&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/lib/rules/indent.js
+++ b/lib/rules/indent.js
@@ -321,7 +321,7 @@ module.exports = function(context) {
             (calleeNode.parent.type === "Property" ||
             calleeNode.parent.type === "ArrayExpression")) {
             // If function is part of array or object, comma can be put at left
-            indent = getNodeIndent(calleeNode, false, true);
+            indent = getNodeIndent(calleeNode, false, false);
         } else {
             // If function is standalone, simple calculate indent
             indent = getNodeIndent(calleeNode);

--- a/tests/lib/rules/indent.js
+++ b/tests/lib/rules/indent.js
@@ -998,6 +998,27 @@ ruleTester.run("indent", rule, {
             "}",
             options: [4, {"VariableDeclarator": 1, "SwitchCase": 1}],
             parserOptions: { ecmaVersion: 6 }
+        },
+        {
+            code:
+            "var a = {\n" +
+            "  some: 1\n" +
+            ", name: 2\n" +
+            "};\n",
+            options: [2]
+        },
+        {
+            code:
+            "a.c = {\n" +
+            "    aa: function() {\n" +
+            "        'test1';\n" +
+            "        return 'aa';\n" +
+            "    }\n" +
+            "    , bb: function() {\n" +
+            "        return this.bb();\n" +
+            "    }\n" +
+            "};\n",
+            options: [4]
         }
     ],
     invalid: [

